### ISSUE TITLE
inline language code highlight

### ISSUE THREAD

**Issue by kwladyka** [yogthos/markdown-clj#96]
Hi,

I am using cryogen and want to highlight also inline code.

Like for example i want write:
So what `(swap! foo-atom inc)` really do?

Why no option to show that as Clojure code like

```clojure
(swap! foo-atom inc)
```

Can we add something like that? How can i achieve that?

**Comment by yogthos** [comment]
There isn't any standard markdown syntax for doing inline language hints. One option would be to make a custom transformer to handle that case.

**Comment by kwladyka** [comment]
I found this link http://stackoverflow.com/questions/23226224/inline-code-syntax-highlighting-in-github-markdown

Do you think `(def foo "bar")`{clojure} or something like that make sense for your repository? Or i should care about that on my own independent?

**Comment by yogthos** [comment]
Yeah, I'd be ok with using GitHub flavored md extension. If you're up for doing a pr, I can help with that. Otherwise, I'll take a look when I have some free time.

**Comment by kwladyka** [comment]
Great. I will do it also in my free time :) So we will see who will be first :)

**Comment by yogthos** [comment]
Sounds like a plan :-)
